src/views/schedule.jade

```diff
@@ -1,4 +1,152 @@
 extends layout
 
 block content
-	h1(style="text-align:center;") Coming Soon
+	table
+		tr
+			th Home
+			th Away
+			th Date
+			th Time
+			th Result
+		tr
+			td.rangers
+				img(src = "/favicon-32x32.png")
+				span  Minnesota Rangers
+			td
+				img(src = "/broncos-32x32.png")
```

Comment:
I wasn't quite sure what to do with those. I'll fix that this afternoon.
On Jan 25, 2016 8:41 AM, "Jake Moening" <notifications@github.com> wrote:

> In src/views/schedule.jade
> <https://github.com/unsignedbytes/mnrangers/pull/21#discussion_r50692733>:
>
> > @@ -1,4 +1,152 @@
> >  extends layout
> >
> >  block content
> > -	h1(style="text-align:center;") Coming Soon
> > +	table
> > +		tr
> > +			th Home
> > +			th Away
> > +			th Date
> > +			th Time
> > +			th Result
> > +		tr
> > +			td.rangers
> > +				img(src = "/favicon-32x32.png")
> > +				span  Minnesota Rangers
> > +			td
> > +				img(src = "/broncos-32x32.png")
>
> Can you put the other team icons into a subfolder in images.
> images/otherLogos/ or something like that. I'd like to keep the root folder
> clean of non-ranger stuff if possible
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/unsignedbytes/mnrangers/pull/21/files#r50692733>.
>